src/pages/execution/tabLoader.tsx

```diff
@@ -35,6 +37,10 @@ type ITileFactoryContext = {
   dag: WorkflowDAG;
   selectedTask: TaskCoordinate | null;
   setSelectedTask: (selectedTask: TaskCoordinate | null) => void;
+  severity: Severity;
```

Comment:
These variables should not be part of `TaskSelection`. Try looking at`ITileFactoryContext`.